lib/Heartbeat/LED.cpp

```diff
@@ -56,11 +58,13 @@ void LED::setState(bool state) {
   }
 }
 
-bool LED::getState() { return digitalRead(_pin) ^ !_activeHigh; }
+bool LED::getState() {
+  return static_cast<bool>(digitalRead(_pin) ^ !_activeHigh);
+}
 
-LED::~LED() {}
+LED::~LED() = default;
```

Comment:
now it is `default`. Why ever?

Reply:
You can probably squash it together. Default is the better option.

Reply:
Lets squash.

Reply:
done.